install/k8s/k8s1.6/contiv.yaml

```diff
@@ -273,88 +260,17 @@ spec:
                 configMapKeyRef:
                   name: contiv-config
                   key: config
-          securityContext:
```

Comment:
netmaster does not need to be a privileged container. It has k8s 1.6 RBAC to ask for the privileges it needs.